server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -457,7 +459,7 @@ private synchronized IndexService createIndexService(IndexCreationContext indexC
             bigArrays,
             threadPool,
             indicesQueryCache,
-            n -> schemas.getTranslogIndexer(n),
```

Comment:
`schemas.getTranslogIndexer` can be removed? It was the only place where it was used.